src/components/atoms/SfLabel/SfLabel.html

```diff
@@ -0,0 +1,7 @@
+
+<component
```

Comment:
This label component I have copied and refactor from alpaca storybook.
I have thought that there have some use case using dynamic tag.

But in my opinion, using just label maybe better because it have to be simpler.
I will fix it. 😆

Reply:
Okay. Imho label is better too :)